sql/catalyst/src/main/scala-2.12/org/apache/spark/sql/catalyst/util/CaseInsensitiveMap.scala

```diff
@@ -42,7 +42,10 @@ class CaseInsensitiveMap[T] private (val originalMap: Map[String, T]) extends Ma
   override def +[B1 >: T](kv: (String, B1)): CaseInsensitiveMap[B1] = {
     new CaseInsensitiveMap(originalMap.filter(!_._1.equalsIgnoreCase(kv._1)) + kv)
   }
-
+  
+  override def updated[B1 >: T](key: String, value: B1): CaseInsensitiveMap[B1] =
```

Comment:
I remember we have two copy of this for Scala 2.13 compatibility. Are you sure this is the right place to fix?

Reply:
I replaced `updated` with `+`. So I removed this `updated` here.

I had to override `+` in `2.13` version of `CaseInsensitiveMap`, because a standard `Map` would be returned from `+`.